docs/user_guide.md

```diff
@@ -556,76 +531,184 @@ If you are using a self-signed cert, make sure to copy the CA cert into ```/etc/
 
-Once the database is ready, an overall database updated timestamp will be shown in the **'Vulnerability'** tab of **'Configuration'** section under **'Administration'**. 
+<a id="pluggable-scanners"></a>
+### Connect Harbor to Additional Vulnerability Scanners
+
+To connect Harbor to additional vulnerability scanners, you must have enabled the default Clair scanner when you deployed Harbor. You must install and configure an instance of the additional scanner according to the scanner vendor's requirements. The scanner must expose an API endpoint to Harbor that permits image pushes and pulls. You can deploy multiple different scanners, and multiple instances of the same type of scanner.
+
+1. Log in to the Harbor interface with an account that has Harbor administrator privileges.
+1. Expand **Administration**, and select **Interrogation Services**. 
+   ![Interrogation Services](img/interrogation-services.png)
+1. Click the **New Scanner** button.
+1. Enter the information to identify the scanner.
+   - A name for this scanner instance, to display in the Harbor interface.
+   - An optional description of this scanner instance.
+   - The address of the API endpoint that the scanner exposes to Harbor.
+   ![Add scanner](img/add-scanner.png)
+1. Select how to connect to the scanner from the **Authorization** drop-down menu.
+   ![Select scanner authentication method](img/scanner-auth.png)
+   - **None**: The scanner allows all connections without any security.
+   - **Basic**: Enter a username and password for an account that can connect to the scanner.
+   - **Bearer**: Paste the contents of a bearer token in the **Token** text box.
+   - **APIKey**: Paste the contents of an API key for the scanner in the **APIKey** text box.
+1. Optionally select **Skip certificate verification** if the scanner uses a self-signed or untrusted certificate. 
+1. Optionally select **Use internal registry address** if the scanner should connect to Harbor using an internal network address rather its external URL.
+1. Click **Test Connection** to make sure that Harbor can connect successfully to the scanner. 
+   ![Test scanner connection](img/test-scanner-connection.png)
+1. Click **Add** to connect Harbor to the scanner.
+1. Optionally repeat the procedure to add more scanners.
+1. If you configure multiple scanners, select one and click **Set as Default** to designate it as the default scanner.
+
+### Updating the Vulnerability Metadata in the Database
+
+Vulnerability scanners depend on the vulnerability metadata to complete the analysis process. After the first initial installation, the vulnerability scanner automatically starts to update the metadata database from different vulnerability repositories. The database update might take a while, based on the data size and network connection. 
+
+Once the database is ready, the timestamp of the last update is shown in the **Interrogation Services** > **Vulnerability** tab. 
```

Comment:
>Once the database is ready, the timestamp of the last update is shown in the Interrogation Services > Vulnerability tab.

One more thing we should mention here, not all the scanners support providing such labels. (so far, Clair and anchore scanners provide such info)

Reply:
@steven-zou I reworded this to "Depending on the scanner that you use, once the database is ready, the timestamp of the last update is shown in the **Interrogation Services** > **Vulnerability** tab. Currently, only Clair and Anchore provide timestamp information."